README.md

```diff
@@ -38,14 +38,17 @@ New commands should go under commands.
 Ensure you are running go 1.7 or later.
 
 Install glide first [it's on GitHub](https://github.com/Masterminds/glide)
+```
+go get github.com/Masterminds/glide
```

Comment:
@maleck13 We discussed this yesterday and you mentioned this wasn't necessary, just looking at it again, I still think it might be required? How can we run `glide install` before we've installed glide?